doc/release/0.2.rst

```diff
@@ -1,6 +1,37 @@
 ***********************
-ARMI v0.2 Release Notes
+ARMI v0.3 Release Notes
+=======================
+
+ARMI v0.3.1
+============
+Release Date: TBD
```

Comment:
Whoops!

It looks like you accidentally over-wrote our `0.2` release notes file.  We have a totally different file for the `0.3` release notes.

I don't want to LOSE the old release notes, can you move this to the existing `doc/0.3.rst` files?

Thanks!

Reply:
Whoops, fixed! https://github.com/terrapower/armi/pull/1425/commits/c2cd59d54051d93b744c0c49e7912eab6b5458c1